pandas/_libs/tslibs/src/datetime/pd_datetime.c

```diff
@@ -53,6 +53,7 @@ static int pandas_datetime_exec(PyObject *module) {
   capi->get_datetime_iso_8601_strlen = get_datetime_iso_8601_strlen;
   capi->make_iso_8601_datetime = make_iso_8601_datetime;
   capi->make_iso_8601_timedelta = make_iso_8601_timedelta;
+  capi->hash_datetime_from_struct = hash_datetime_from_struct;
```

Comment:
You also need to define a macro for this later in the file

Reply:
```
#define hash_datetime_from_struct(dts)                                         \
  PandasDateTimeAPI->hash_datetime_from_struct((dts))
```
 didnt seem to do it